go-sdk/pkg/sdk/mock_eventsource.go

```diff
@@ -0,0 +1,80 @@
+package sdk
+
+import (
+	"context"
+	"fmt"
+	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/keptn/cp-connector/pkg/controlplane"
+	"sync"
+)
+
+func NewTestEventSource() *TestEventSource {
```

Comment:
should we move the mock implementations to `pkg/sdk/mock`?

Reply:
I've tried it. unfortunately It results in a circular dependency for m̀ock_taskhandler.go` :(

Reply:
turned out that i don't need the mocks anymore at all :D